resources/type-system/strict-inference.md

```diff
@@ -171,7 +171,9 @@ type cannot be inferred from downwards inference (in the case of a function
 literal) is considered an inference failure. A function literal's parameter
 types are commonly inferred when assigning the literal to a variable typed with
 a typedef, or when passing the literal as an argument whose corresponding
-parameter is  function-typed.
+parameter is function-typed.
+
+"Unused" parameters named `_`, `__`, etc. never represent an inference failure.
```

Comment:
What does quote-*unused*-unquote mean?
Must the parameters actually be unused or does it apply to any parameters named `_`*? In the former case, remove the quotes, and if it's the latter then remove the `"Unused"` entirely, and perhaps add a rationale text like:
> *Such names traditionally represent variables which are ignored, and as such, their type, `dynamic` or not, is not important.*

Reply:
Good clarification; it's the latter. Done.